homeworks/andrii.tkachenko_Hodr1337/4-objects/objects.js

```diff
@@ -0,0 +1,72 @@
+const readline = require('readline');
+
+const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+});
+
+rl.question('Please write the actions from the list (heal,play,feed) that you want to do with tamagotchi using the space separator. Example : heal play feed. Write it.. ', (answer) => {
+    const userAnswer = answer || 'heal play feed';
+
+    const re = /\s* \s*/;
+    const yourActionsArray = userAnswer.split(re);
+
+    const tamagochi = {
+        health: 5,
+        enjoy: 5,
+        food: 5,
+    };
+
+    function healIt() {
+        tamagochi.health += 3;
+        if (tamagochi.health >= 10) tamagochi.health = 10;
+
+        tamagochi.enjoy -= 1;
+        if (tamagochi.enjoy <= 0) console.log('Opps. Your tamagochi died for lack of happiness');
+
+        tamagochi.food -= 1;
+        if (tamagochi.food <= 0) console.log('Opps. Your tamagochi died of hunger');
+    }
+
+    function playIt() {
+        tamagochi.health -= 1;
+        if (tamagochi.health <= 0) console.log('Opps. Your tamagochi died from lack of health');
+
+        tamagochi.enjoy += 3;
+        if (tamagochi.enjoy >= 10) tamagochi.enjoy = 10;
+
+        tamagochi.food -= 1;
+        if (tamagochi.food <= 0) console.log('Opps. Your tamagochi died of hunger');
+    }
+
+    function feedIt() {
+        tamagochi.health -= 1;
+        if (tamagochi.health <= 0) console.log('Opps. Your tamagochi died from lack of health');
+
+        tamagochi.enjoy -= 1;
+        if (tamagochi.enjoy <= 0) console.log('Opps. Your tamagochi died for lack of happiness');
+
+        tamagochi.food += 3;
+        if (tamagochi.food >= 10) tamagochi.food = 10;
+    }
+
+    yourActionsArray.forEach((element) => {
+        if (element === 'heal') {
+            // eslint-disable-next-line no-param-reassign
+            element = healIt();
+        }
+        if (element === 'play') {
+            // eslint-disable-next-line no-param-reassign
+            element = playIt();
+        }
+        if (element === 'feed') {
+            // eslint-disable-next-line no-param-reassign
+            element = feedIt();
+        }
+    });
```

Comment:
@Hodr1337 hey!
You don't need to reassign the element. Instead, you need to call the function.
Also, you can use switch-case here instead of if.